File: Service/Implement/DanhMucTieuChuanService.cs
namespace Service.Implement
{
    public class DanhMucTieuChuanService : BaseService<DanhMucTieuChuan, IDanhMucTieuChuanRepository>
    , IDanhMucTieuChuanService
    {
    private readonly IDanhMucTieuChuanRepository _DanhMucTieuChuanRepository;
    public DanhMucTieuChuanService(IDanhMucTieuChuanRepository DanhMucTieuChuanRepository) : base(DanhMucTieuChuanRepository)
    {
    _DanhMucTieuChuanRepository = DanhMucTieuChuanRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucTieuChuan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucTieuChuan } from 'src/app/shared/DanhMucTieuChuan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucTieuChuanService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucTieuChuan";
    }
}

